samtranslator/model/eventsources/push.py

```diff
@@ -17,6 +17,8 @@
 
 CONDITION = 'Condition'
 
+RequestParameterProperties = ["Required", "Caching"]
```

Comment:
Updated to all caps with underscores.  I left this outside the class because I followed a similar pattern from https://github.com/awslabs/serverless-application-model/blob/master/samtranslator/model/api/api_generator.py#L27, which btw does not follow that same convention of all caps with underscores.

Reply:
Ah apologies for the inconsistency there, but thanks for fixing this one!